code/modules/mob/living/carbon/human/species/station/alraune.dm

```diff
@@ -0,0 +1,336 @@
+	breath_cold_level_3 = 140 //mainly 'cause breath tolerance is more generous than body temp tolerance.
+
+	breath_heat_level_1 = 400 //slightly better heat tolerance in air though. Slightly.
+	breath_heat_level_2 = 450
+	breath_heat_level_3 = 800 //lower incineration threshold though
+
+	spawn_flags = SPECIES_CAN_JOIN | SPECIES_IS_WHITELISTED // whitelist only while WIP
+	flags = NO_SCAN | IS_PLANT | NO_MINOR_CUT
+	appearance_flags = HAS_HAIR_COLOR | HAS_LIPS | HAS_UNDERWEAR | HAS_SKIN_COLOR | HAS_EYE_COLOR
+
+	inherent_verbs = list(
+		/mob/living/carbon/human/proc/succubus_drain,
+		/mob/living/carbon/human/proc/succubus_drain_finalize,
+		/mob/living/carbon/human/proc/succubus_drain_lethal,
+		/mob/living/carbon/human/proc/bloodsuck) //Give them the voremodes related to wrapping people in vines and sapping their fluids
+
+	color_mult = 1
+	icobase = 'icons/mob/human_races/r_human_vr.dmi'
+	deform = 'icons/mob/human_races/r_def_human_vr.dmi'
+	flesh_color = "#9ee02c"
+	blood_color = "#edf4d0" //sap!
+	base_color = "#1a5600"
+
+	blurb = "Alraunes are a rare sight in space. Their bodies are reminiscent of that of plants, and yet they share many\
+	traits with other humanoid beings.\
+	\
+	Most Alraunes are not interested in traversing space, their heavy preference for natural environments and general\
+	disinterest in things outside it keeps them as a species at a rather primal stage.\
+	\
+	However, after their discovery by the angels of Sanctum, many alraunes succumbed to their curiosity, and took the offer\
+	to learn of the world and venture out, whether it's to Sanctum, or elsewhere in the galaxy."
+
+	has_limbs = list(
+		BP_TORSO =  list("path" = /obj/item/organ/external/chest),
+		BP_GROIN =  list("path" = /obj/item/organ/external/groin),
+		BP_HEAD =   list("path" = /obj/item/organ/external/head),
+		BP_L_ARM =  list("path" = /obj/item/organ/external/arm),
+		BP_R_ARM =  list("path" = /obj/item/organ/external/arm/right),
+		BP_L_LEG =  list("path" = /obj/item/organ/external/leg),
+		BP_R_LEG =  list("path" = /obj/item/organ/external/leg/right),
+		BP_L_HAND = list("path" = /obj/item/organ/external/hand),
+		BP_R_HAND = list("path" = /obj/item/organ/external/hand/right),
+		BP_L_FOOT = list("path" = /obj/item/organ/external/foot),
+		BP_R_FOOT = list("path" = /obj/item/organ/external/foot/right)
+		)
+
+	// limited organs, 'cause they're simple
+	has_organ = list(
+		O_LIVER =    /obj/item/organ/internal/liver/alraune,
+		O_KIDNEYS =  /obj/item/organ/internal/kidneys/alraune,
+		O_BRAIN =    /obj/item/organ/internal/brain/alraune,
+		O_EYES =     /obj/item/organ/internal/eyes/alraune,
+		)
+
+/datum/species/alraune/can_breathe_water()
+	return TRUE //eh, why not? Aquatic plants are a thing.
+
+
+/datum/species/alraune/handle_environment_special(var/mob/living/carbon/human/H)
+	if(H.inStasisNow()) // if they're in stasis, they won't need this stuff.
+		return
```

Comment:
This is like 200 lines of copypaste? There's no better way?

Reply:
I'd love to just use handle_breath() from line 132 onwards, but there's a whole lot of stuff in handle_breath() that's really bad for them, like making them gasp when there's oxygen present because it's their exhale type, damage from hot air specifically targeting their head and giving messages about icicles in the lungs, and so on. Given that they're plants and don't have lungs I'd honestly be fine with massively simplifying their breath code here though.